en/contribute/maintainers.md

```diff
@@ -0,0 +1,93 @@
+
+To find the most up-to-date maintainers structure, visit [PX4-Autopilot README](https://github.com/PX4/PX4-Autopilot#maintenance-team).
+
+## Maintainers Call
+
+We have a weekly maintainers call to discuss overall PX4 structure, community and ecosystem. You can find the relevant information below:
+
+* Time: Tuesday 17h00 CET, 11h00 EST, 08h00 PST ([subscribe to calendar](https://www.dronecode.org/calendar/))
+* Place: [Dev-Call-Maintainers channel in Discord](https://discord.gg/RjxYXFSnj3)
+* Meeting note is published week before in [Discuss Forum - PX4 Coordination](https://discuss.px4.io/c/weekly-dev-call/px4-dev-call/39), and users are encouraged to add the agendas as a reply before the meeting starts.
+* To nominate Issues and PRs for the call you can use the [maintainers-call](https://github.com/PX4/PX4-Autopilot/labels/maintainers-call) label to flag them
+
+## Recruitment process
+
+If you would like to join the PX4 Maintainers Team, please do the following:
+
+1. Read the [role description](#dronecode-maintainer-role-description), and make sure you understand the responsibilities & your role.
+2. Contact the relevant maintainer listed in the [PX4-Autopilot README](https://github.com/PX4/PX4-Autopilot#maintenance-team) on [Discord server](support.md#forums-and-chat), or any of the preferred method.
+3. Express your interest in becoming a maintainer, and specify which sector you would like to maintain.
+4. Bring this up in the [weekly maintainers call](#maintainers-call), and if there's a general consensus on approval, you will become a maintainer!
+5. If approved, go through the [onboarding process](#onboarding-process)
+
+## Onboarding process
+
+After you become a maintainer, you will go through the following process:
+
+1. **Discord** server admin will grant you the `dev team` role, which gives you:
+   1. Permission to speak during maintainers call
+   2. Access to the `#maintainers` channel
+   3. Access to the user group / dronecode member companies channels
+2. **Github** PX4 Workspace admin will grant you the [`Dev Team`](https://github.com/orgs/PX4/teams/dev-team) role, which gives you:
+   1. Permission to merge the PR of any of PX4 workspace repositories after it's approved
+   2. Permission to trigger Github actions when a new contributor opens a PR
+   3. Permission to edit Issue/PR contents
+3. **Dronecode member** (Junwoo/Ramon) will:
+   1. Update the PX4 code repository [README](https://github.com/PX4/PX4-Autopilot/blob/main/README.md) to include your information
+   2. Update the PX4 website [Maintainers section](https://px4.io/community/maintainers/) to include your information
+   3. Add your information to the internal Dronecode database of maintainers to keep you in sync
+   4. Create a social media post about your new maintainers role after a short interview
```

Comment:
I have defined the clear steps of the recruitment & onboarding process here @mrpollo